scripts/upgrade_database.sh

```diff
@@ -13,7 +13,7 @@ if [ -z "${ZEN_ENV:-}" ] ; then
   exit 1
 fi
 
-ZEN_SRC_ROOT=$(git rev-parse --show-toplevel)
+ZEN_SRC_ROOT=$(dirname $(dirname $(readlink -f "$0")))
```

Comment:
@jbinary can you think of a nicer way of doing this? (I just want the parent directory that the script is in)

Reply:
You can replace that second `dirname` with `/..` instead after `readlink` but I don't even think it's really better